src/objecttypes/conf/api.py

```diff
@@ -1,6 +1,6 @@
 from vng_api_common.conf.api import *  # noqa - imports white-listed
 
-API_VERSION = "2.2.2"
+API_VERSION = "3.0.0"
```

Comment:
[This](https://github.com/maykinmedia/objects-api/pull/517#discussion_r1910020235) applies here too

Reply:
I don't think we need to increase this to v3+ right now, since we still use the v2 endpoint. This would increase to 3.0.0+ if we actually make breaking changes to the API itself. 

I don't think we've been updating this version the right way though, I guess technically it should be increased dependent on whether or not there have been major/minor/patch changes to the API. Since this release doesn't change the v2 API, I think we should leave this number unchanged.